pkgs/by-name/li/lib60870/package.nix

```diff
@@ -0,0 +1,28 @@
+{ cmake
+, lib
+, stdenv
+, fetchzip
+}:
+stdenv.mkDerivation {
+  pname = "lib60870";
+  version = "2.3.1";
```

Comment:
Newest version is 2.3.2 from https://github.com/mz-automation/lib60870/releases

Reply:
Thanks. Addressed in e1370a466ca9174fe8ba5f699321d4db48dfefb4